.byebug_history

```diff
@@ -0,0 +1,7 @@
+quit
+c
+current_user.missing_organization?
+c
+current_user.organization
+c
+current_user.organization_id
```

Comment:
Please do not commit your byebug history. Add the file to your project's `.gitignore`. 😎 👉 👉